sklearn/multiclass.py

```diff
@@ -203,6 +213,49 @@ def fit(self, X, y):
 
         return self
 
+    def partial_fit(self, X, y, classes=None):
+        """Partially fit underlying estimators
+        
+        Should be used when memory is inefficient to train all data.
+        Chunks of data can be passed in several iteration, where the
+        first call should have an array of all target variables.
+        
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+        y : (sparse) array-like, shape = [n_samples] or [n_samples, n_classes]
+            Multi-class targets. An indicator matrix turns on multilabel
+            classification.
+        Returns
+        -------
+        self
+        """
+        if self._first_partial_run:
+            if classes is None:
+                raise ValueError("array of all classes of target varialble"
+                                  "must be provided")
```

Comment:
I think that the discoverability of the classes argument should be improved:
  * the error message above should make it clear that the classes argument is only for the first call
  * the docstring of fit should mention this argument, and its role (as eg in the docstring of the SGDClassifier).